src/main/java/org/jabref/gui/edit/CopyMoreAction.java

```diff
@@ -222,9 +223,11 @@ private void copyKeyAndLink() {
             String url = entry.getField(StandardField.URL).orElse("");
             keyAndLink.append(url.isEmpty() ? key : String.format("<a href=\"%s\">%s</a>", url, key));
             keyAndLink.append(OS.NEWLINE);
+            fallbackString.append(url.isEmpty() ? key : String.format("%s - %s", key, url));
+            fallbackString.append(OS.NEWLINE);
         }
 
-        clipBoardManager.setHtmlContent(keyAndLink.toString());
+        clipBoardManager.setHtmlContent(keyAndLink.toString(),fallbackString.toString());
```

Comment:
Please fix checkstyle

```suggestion
        clipBoardManager.setHtmlContent(keyAndLink.toString(), fallbackString.toString());
```

Reply:
Hi Koppor,
I implemented your change requests!
One question:
How can check the "checkstyle" myself?